build.gradle

```diff
@@ -22,7 +22,7 @@ ext {
     def correctVersion = project.hasProperty('versionNumber') && property('versionNumber') ==~ /\d+\.\d+\.\d+.*/
     versionNumber = correctVersion ? property('versionNumber') : 'SNAPSHOT-' + new Date().format('yyyyMMddHHmmss')
     projectIds = ['group': 'teamcity-azure-plugin', 'version': versionNumber]
-    teamcityVersion = findProperty('teamcityVersion') ?: '2020.2'
```

Comment:
we'll need to ensure that plugin will continue working with TC versions older than 2022.10. I'd rather leave it 2020.2 or 2021.1

Reply:
returned 2020.2